src/NServiceBus.AcceptanceTests/Routing/When_using_instance_ids.cs

```diff
@@ -37,20 +38,31 @@ public class UnawareSender : EndpointConfigurationBuilder
         {
             public UnawareSender()
             {
-                EndpointSetup<DefaultServer>(c => { c.Routing().RouteToEndpoint(typeof(MyMessage), GetReceiverEndpoint()); });
+                EndpointSetup<DefaultServer>((c,r) =>
+                {
+                    c.UseTransport(r.GetTransportType()).Routing().RouteTo(typeof(MyMessage), GetReceiverEndpoint());
+                });
             }
         }
 
         public class AwareSender : EndpointConfigurationBuilder
         {
             public AwareSender()
             {
-                EndpointSetup<DefaultServer>(c =>
+                EndpointSetup<DefaultServer>((c, r) =>
                 {
-                    c.Routing().RouteToEndpoint(typeof(MyMessage), GetReceiverEndpoint());
-                    c.Routing().Mapping.Physical.Add(new EndpointInstance(GetReceiverEndpoint(), "XYZ"));
+                    c.UseTransport(r.GetTransportType()).Routing().RouteTo(typeof(MyMessage), GetReceiverEndpoint());
+                    c.EnableFeature<SpecificRoutingFeature>();
                 });
             }
+
+            class SpecificRoutingFeature : Feature
```

Comment:
seems weird to have a feature to just setup up the instance? wouldn't that be something you'd want to setup on the `EndpointSetup`?